### DISCUSSION TITLE
wx.lin弃用后调用瀑布流renderWaterFlow方法问题

### DISCUSSION THREAD

**Discussion by Leo-snow** [TaleLin/lin-ui#1355] [求助] [answered]
wx.lin可使用时瀑布流渲染方法加载：

>     wx.lin.renderWaterFlow(this.data.demo, false, () => {  console.log('渲染成功')  })

请问不用wx.lin该如何调用

**Comment by juzi214032** [comment, marked as answer]
瀑布流组件暂时还未增加开放函数，后续版本会逐渐添加
